docs/dev-standards/secure-coding-and-tooling.md

```diff
@@ -0,0 +1,67 @@
+# Secure coding and tooling
+
+## Introduction
+
+Secure coding and automated tooling help teams build software that is safe, reliable and maintainable. By detecting issues early in the development process, teams can reduce the cost of remediation and improve delivery confidence.
+
+This guidance sets out the minimum expectations for secure coding practices and the use of automated tools in engineering teams.
+
+## Guidance
+
+Teams **MUST**:
+
+- use static analysis tools to detect code quality and security issues before merging code
+- prevent secrets from being committed to source control
+- perform Software Composition Analysis (SCA) to identify supply chain risks
+- enforce security checks in CI/CD pipelines for all branches
+
+Teams **SHOULD**:
+
+- use tools such as [SonarCloud][1], [Snyk][2], [CodeQL][5], and [detect-secrets][3] to automate scanning
```

Comment:
I think SonarCloud and Snyk are likely to be mandated. Check with Graham to ensure this aligns with his plans.